composer.json

```diff
@@ -14,6 +14,7 @@
     "composer/installers": "~1.0"
   },
   "require-dev": {
+    "automattic/vipwpcs": "@stable",
```

Comment:
Could we add the version following the SEMVER syntax similar to the rest of the dependencies. This would also ensure that Dependabot can open pull requests for dependency upgrade and we're keeping the same style for clarify.

Reply:
`"^2.0.0"` should do it